visualization/app/codeCharta/ui/legendPanel/legendPanel.component.ts

```diff
@@ -133,16 +142,13 @@ export class LegendPanelController implements SettingsServiceSubscriber {
 		return pathArray[pathArray.length - 1]
 	}
 
-	private initAnimations() {
-		$(document).ready(() => {
-			let start = 40
-			let target = -500
-			let visible = false
-			$("legend-panel-component .panel-button").click(() => {
-				$("legend-panel-component .block-wrapper").animate({ right: visible ? target + "px" : start + "px" }, "fast")
-				visible = !visible
-			})
-		})
+	public toggle() {
```

Comment:
Is there a reason we don't use ng-class or similar instead of this?

Reply:
What exactly do you think of?

Reply:
essentially instead of using JQuery, i used angularjs ng-class to add/remove the class based on an attribute in the viewModel. Personally I think it is nicer, if you don't like it you can revert the two commits in question ;)

Reply:
actually I like it a lot! Thanks :)